app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -43,6 +43,7 @@
     public static final int IMAGE_WAIT = -2;
     public static final int EMPTY_TITLE = -3;
     public static final int FILE_NAME_EXISTS = -4;
+    public static final int FILE_FBMD = -5;
```

Comment:
Javadoc is probably needed here. Sorry that the other values have no Javadoc, but it is not to late to start doing things the good way :-)

Reply:
I have added the Javadoc for the variable